modulus/models/module.py

```diff
@@ -273,10 +273,13 @@ def load(
 
         # Download and cache the checkpoint file if needed
         cached_file_name = _download_cached(file_name)
+        print("load cached_file_name: ", cached_file_name)
```

Comment:
Could you please remove the rest of the added print statements in this file as well?

Reply:
removed all inconsistent print XXX lines.